### ISSUE TITLE
It seems when scraping a repeated filename, the first file remains cached 

### ISSUE THREAD

**Issue by kenarab** [ropenscilabs/tabulizer#26]
I was scraping repeated filenames in different folders and when I execute extract_table and/or extract_areas it returns the first instance of the filename.

For me worked a workaround: load again the library.

**Comment by kenarab** [comment]
Nope. Is not working. I have to restart the session for it works ok with duplicated filenames in different folders.

**Comment by leeper** [comment]
Sorry, I'm not following exactly. Can you give an example that demonstrates the issue?
